airgun/views/contenthost.py

```diff
@@ -79,6 +79,20 @@ def read(self):
         return self.color
 
 
+class ModuleStreamActionDropDown(ActionsDropdown):
+    """Custom drop down for module streams tab in content host page"""
+    customize_check_box = Checkbox(id="customize")
```

Comment:
`is_customize` should be mandatory attribute, so we have exception if user provide not the expected data
```
    customize_check_box = Checkbox(id="customize")

    def fill(self, item):
        self.open()
        customize_check_box.fill(item['is_customize])
        self.select(item['action'])
```